proto/options.proto

```diff
@@ -87,6 +87,23 @@ message HierarchyLimits {
                                    // always allowed. Used for A*.
 }
 
+// for /tile requests
+message Tile {
+  oneof has_x {
```

Comment:
we simulate optionality via this `oneof`? Do we have a usecase when it is valid request and one of x/y/z are not passed? 🤔